.changeset/poor-jokes-hear.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/catalog-backend-module-okta': patch
+---
+
+Allow configuration to allow some crude throttling of Okta group membership requests.
```

Comment:
Looks to be just chunking here instead of actually throttling the endpoint calls. Is that expected and should we change to changeset to match?

Reply:
I think I have used chunking to effectively limit the number of api calls that are sent to okta, which in my mind is throttling. Am I getting something wrong? Ill add some more detail here.

Reply:
I have changed the message here. Let me know if it looks better?